src/shared/lib/alerts/permissionAlerts.ts

```diff
@@ -3,29 +3,28 @@ import { Alert } from 'react-native';
 import i18n from 'i18next';
 import { openSettings } from 'react-native-permissions';
 
-export const handleMicrophoneBlockedPermissions = async () =>
+export const handleBlockedPermissions = async (
+  title: string,
+  subtitle: string,
+) =>
   new Promise(resolve => {
     const { t } = i18n;
 
-    Alert.alert(
-      t('audio_recorder:alert_title'),
-      t('audio_recorder:alert_message'),
-      [
-        {
-          text: t('audio_recorder:alert_button_cancel'),
-          onPress: () => {
-            resolve(false);
-          },
-          style: 'cancel',
+    Alert.alert(title, subtitle, [
+      {
+        text: t('audio_recorder:alert_button_cancel'),
```

Comment:
In case we make a generic function that handles multiple cases it is better to create separate entries in the translation file for these button texts. So it should be look like t('permissions:alert_button_cancel')